src/darwin/Framework/CHIP/zap-generated/MTRCommandPayloadsObjc.h

```diff
@@ -11072,9 +11072,9 @@ MTR_PROVISIONALLY_AVAILABLE
 MTR_AVAILABLE(ios(18.4), macos(15.4), watchos(11.4), tvos(18.4))
 @interface MTRContentAppObserverClusterContentAppMessageParams : NSObject <NSCopying>
 
-@property (nonatomic, copy) NSString * _Nullable data MTR_AVAILABLE(ios(18.4), macos(15.4), watchos(11.4), tvos(18.4));
+@property (nonatomic, copy) NSString * _Nonnull data MTR_AVAILABLE(ios(18.4), macos(15.4), watchos(11.4), tvos(18.4));
```

Comment:
This is an API break, as is the corresponding change in Commands.h... 

See also https://github.com/project-chip/connectedhomeip/issues/32669 where this discrepancy wrt spec was raised and then rather ignored...